### ISSUE TITLE
[Enhancement]: Update styling of SingleEvent to something more upbeat and lively.

### ISSUE THREAD

**Issue by manuel12** [gbowne1/reactsocialnetwork#265]
### Describe the Bug

@gbowne1 @pawel975 

The current general styling of the app is super grey and not much fun. I suggest we update some components to add more color.

I wanna start with this design for the SingleEvent components:

![image](https://github.com/gbowne1/reactsocialnetwork/assets/4129325/477ea737-e897-4a2f-9374-d0c36766a963)

This is a rough estimate and the final product colors might look different and I also  have to consider dark mode colors - but I think the idea is there.

### Steps to Reproduce

Run the app - go to events section

### Which device are you using?

Asus VivoBook

### Which operating system are you using?

Windows

### Which browser are you using?

Chrome

### Contribute

- [X] I am willing to contribute and submit a pull request

**Comment by gbowne1** [comment]
great idea @manuel12 :-) I think we could build a colorized theme too that the user could select in the settings or in some other way.

**Comment by manuel12** [comment]
@gbowne1 @pawel975 

So I finished adding new styles to SingleEvents. This is how they look now on light mode:

![image](https://github.com/gbowne1/reactsocialnetwork/assets/4129325/f2ccbed0-69c8-4604-929d-b921f1d8ad49)

And on dark mode:

![image](https://github.com/gbowne1/reactsocialnetwork/assets/4129325/b8620ed1-b8a1-4bf5-ad26-325ae7a5ba61)

I forgot to create a branch and committed and push directly to main so there will not be a PR for this one. I ran the e2e tests and they all pass in any case.
